File: ansilove/strtolower.m
//
//  strtolower.m
//  AnsiLove/ObjC
//
//  Copyright (c) 2011, Stefan Vogt. All rights reserved.
//  http://byteproject.net
//
//  Use of this source code is governed by a MIT-style license.
//  See the file LICENSE for details.
//

#import "strtolower.h"

void strtolower(char str[]) {
    while(*str != 0) {
        if (*str >= 'A' && *str <= 'Z') {
            *str = *str + 'a' - 'A';
        }
        str++;
    }
}

File: ansilove/strtolower.c
//
//  strtolower.m
//  AnsiLove/C
//
//  Copyright (C) 2011-2013 Stefan Vogt, Brian Cassidy, Frederic Cambus.
//  All rights reserved.
//  https://github.com/ByteProject/AnsiLove-C/
//
//  Use of this source code is governed by a MIT-style license.
//  See the file LICENSE for details.
//

#if defined(__APPLE__) && defined(__MACH__)
#import "strtolower.h"
#else
#include "strtolower.h"
#endif

char *strtolower(char *str)
{
    char *p;
    for (p = str; *p != '\0'; ++p) 
    {
        *p = tolower(*p);
    }
    return str;
}
